src/modules/modules.py

```diff
@@ -213,8 +217,10 @@ def forward(self, sent, task, reset=True):
             else:
                 sent_enc = torch.cat([sent_enc, skip_vec], dim=-1)
 
+
```

Comment:
Revert this—it's not a good idea to change code that isn't crucial for the PR, and two blank lines within a method is pretty unusual.